syncstorage/tests/functional/test_old_storage.py

```diff
@@ -1,611 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Was this file deleted on purpose? Removing it seems unrelated to the gist of the PR, but as always I could easily be missing something.

Reply:
No, it was by accident. I had renamed the file to aid in testing. It should be restored in the latest push.